app/models/micropost.rb

```diff
@@ -0,0 +1,28 @@
+# Micropost model sample
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  # RuboCop will say "You can use `validates` instead of `validates_*`.".
+  # For example:
+  #   validates :content, presence: true, length: { maximum: 200 }
+  validates :content, presence: true, length: { maximum: 200 }
+
+  def convert_content(pattern, replaced)
+    content.gsub(pattern, replaced)	
+  end
+
+  #
+  # Class method
+  #
+  class << self
+    def convert_content(pattern, replaced)
+      Micropost.transaction do
+        Micropost.all.each do |m|
```

Comment:
> __[RuboCop]__ Rails/FindEach: Use `find_each` instead of `each`. ([link](https://sider.review/gh/repos/171964967/pulls/2#issue-3735186))

こんかいはデータがすくないのでeachで大丈夫！